common/src/main/java/com/wynntils/screens/maps/SeaskipperDepartureBoardScreen.java

```diff
@@ -0,0 +1,402 @@
+
+        this.addRenderableWidget(new SeaskipperBoatButton(
+                (int) (destinationListWidth - boatButtonWidth + 5f),
+                (int) (this.height - boatButtonHeight - 5f),
+                (int) boatButtonWidth,
+                (int) boatButtonHeight));
+
+        float travelButtonWidth = Texture.TRAVEL_BUTTON.width() * currentTextureScale;
+        float travelButtonHeight = Texture.TRAVEL_BUTTON.height() / 2f * currentTextureScale;
+
+        this.addRenderableWidget(new SeaskipperTravelButton(
+                5,
+                (int) (this.height - travelButtonHeight - 5f),
+                (int) travelButtonWidth,
+                (int) travelButtonHeight,
+                this));
+
+        destinationListY =
+                this.height - Texture.DESTINATION_LIST.height() * currentTextureScale - travelButtonHeight - 5f;
+
+        renderY = destinationListY;
+
+        renderHeight = Texture.DESTINATION_LIST.height() * currentTextureScale;
+
+        mapHeight = renderHeight - renderedBorderYOffset * 2f;
+
+        centerZ = renderY + renderedBorderYOffset + mapHeight / 2f;
+    }
+
+    @Override
+    public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialTick) {
+        RenderSystem.setShaderColor(1f, 1f, 1f, 1f);
+
+        RenderSystem.enableDepthTest();
+
+        renderMap(poseStack);
+
+        RenderUtils.enableScissor(
+                (int) (renderX + renderedBorderXOffset), (int) (renderY + renderedBorderYOffset), (int) mapWidth, (int)
+                        mapHeight);
+
+        renderDestinations(
+                poseStack,
+                BoundingBox.centered(mapCenterX, mapCenterZ, width / currentZoom, height / currentZoom),
+                1,
+                mouseX,
+                mouseY);
+
+        renderCursor(
+                poseStack,
+                1.5f,
+                Managers.Feature.getFeatureInstance(CustomSeaskipperScreenFeature.class)
+                        .pointerColor
+                        .get(),
+                Managers.Feature.getFeatureInstance(CustomSeaskipperScreenFeature.class)
+                        .pointerType
+                        .get());
+
+        RenderUtils.disableScissor();
+
+        renderBackground(poseStack);
+
+        renderCoordinates(poseStack, mouseX, mouseY);
+
+        RenderUtils.drawScalingTexturedRect(
+                poseStack,
+                Texture.DESTINATION_LIST.resource(),
+                5,
+                destinationListY,
+                0,
+                destinationListWidth,
+                Texture.DESTINATION_LIST.height() * currentTextureScale,
+                Texture.DESTINATION_LIST.width(),
+                Texture.DESTINATION_LIST.height());
+
+        renderWidgets(poseStack, mouseX, mouseY, partialTick);
+    }
+
+    @Override
+    protected void renderCoordinates(PoseStack poseStack, int mouseX, int mouseY) {
```

Comment:
I would remove coordinate rendering, since having these 3 cases makes it a bit confusing to know what is displayed imo.

Reply:
Could cut it down to just the mouse pos, or entirely remove?

Reply:
I am not sure if that would be useful at all?